### PR TITLE
Addon-Docs: HTML support & example

### PR DESCRIPTION
**Pull request by shilman**
Issue: #7101 

## What I did

- Add HTML preset to addon-docs (that just uses the common one)
- Add DocsPage and MDX example to `html-kitchen-sink`

## How to test

```
cd examples/html-kitchen-sink
yarn storybook
```

Or just check out the deployed storybook below

### CODE REVIEW

**Comment by Hypnosphi** [comment]
I wonder whether we can make JSX work here

File: examples/html-kitchen-sink/stories/addon-docs.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+<Meta title="Addons|Docs" />
+
+## Story definition
+
+Hallelujah! HTML is working out of the box without modification.
+How you like them apples?!
+
+<Story name="heading" height="100px">
+  {'<h1>Hello World</h1>'}
```
Comment (Hypnosphi) [inline examples/html-kitchen-sink/stories/addon-docs.stories.mdx]: I wonder whether we can make JSX work here

Comment (shilman) [reply examples/html-kitchen-sink/stories/addon-docs.stories.mdx]: We probably can. To start with I want to have the `<Story>` contents have a 1-to-1 correspondence to the return function from the story function, for simplicity's sake. In 6.0 (possibly in conjunction with some subset of multiframework and monoconfig work) we can rethink this on a framework-by-framework basis.

Also in the meantime, MDX may evolve to support some of this stuff natively. I know Vue support is in alpha, for example.

**Review by tmeasday — APPROVED** [review]
LGTM

File: examples/html-kitchen-sink/stories/addon-docs.stories.mdx (hunk 1 above)
Comment (Hypnosphi) [reply examples/html-kitchen-sink/stories/addon-docs.stories.mdx]: Vue templates are valid HTML strings so maybe it would be enough

**Pull request merged**